landingAr.html

```diff
@@ -11,17 +11,18 @@
 </head>
 
 <body>
+  <div class="dropdown landing">
+    <button class="dropbtn" id="landingbtn">تغير لغة ↓</button>
+    <div class="landing-content">
+      <a href="landing.html">English</a>
+      <a href="landingAr.html">العربية</a>
```

Comment:
a tag for Arabic file should be first :v:

Reply:
as I said in #80, I don't think that that should matter at all